### DISCUSSION TITLE
Operation type structure

### DISCUSSION THREAD

**Discussion by zipus** [frePPLe/frepple#579] [Q&A] [answered]
How are the following scenarios designed to be created? I can not find any similar approach on the docs.

CASE 1. Sequential start.
Le'ts say I need to do 1 table, so 4 legs are required.
To do the 4 legs, I have 3 operations (cut 1h x leg / sand 1h x leg / paint 1h x leg).
If I have employees and resources available, once I have cut 1 leg I can start sanding 1 and once sanded 1 can be painted. So in 6 hours I have the legs produced and no need to wait 4h to have 4 legs cut, then 4h to have it sanded and 4h to be painted.
Is there anything to config (what is?) to allow this or it's managed automatically by frepple?

CASE 2. Routing with not sequential operations.
I know routing is for sequential, but how to create steps that are not sequential?
Let's say I need to do A -> B/ C -> D, so B is preceded by A, B and C can be done at the same time (not sequentially) and D needs to have B and C done.

**Comment by jdetaeye** [comment, marked as answer]
>>CASE 1. Sequential start.

Overlapping operations are possible with the transfer batch functionality. See https://frepple.com/docs/current/examples/buffer/transfer-batch.php    This feature not heavily used and your mileage may vary...   
If you can keep the model at the aggregate level (ie 1 operation to represent the 3 detailed/overlapping steps), you'll get a plan that is easier to understand and work with.  

You can also check out the offset field https://frepple.com/docs/current/model-reference/operation-materials.php?highlight=offset   This can also be used to define some overlap or delay between operations.  Eg you can define that the item is ready only 1 hour after painting is finished to account for the drying time.

>>CASE 2. Routing with not sequential operations.

Currently this is not possible, but a customer is sponsoring such an enhancement right now. 
We haven't decided yet whether or not to put the feature in the Community Edition.

**Reply by zipus** [reply]
> Currently this is not possible, but a customer is sponsoring such an enhancement right now.
>  We haven't decided yet whether or not to put the feature in the Community Edition.

Any news?

**Comment by jdetaeye** [comment]
Yes, that feature is available in the 7.1 Community Edition.
